CHANGELOG.md

```diff
@@ -209,7 +209,7 @@
 * [#5050](https://github.com/bbatsov/rubocop/issues/5050): Add auto-correction to `Style/ModuleFunction`. ([@garettarrowood][])
 * [#5358](https://github.com/bbatsov/rubocop/pull/5358):  `--no-auto-gen-timestamp` CLI option suppresses the inclusion of the date and time it was generated in auto-generated config. ([@dominicsayers][])
 * [#4274](https://github.com/bbatsov/rubocop/issues/4274): Add new `Layout/EmptyComment` cop. ([@koic][])
-* [#4477](https://github.com/bbatsov/rubocop/issues/4477): Add new configuration directive: `inherit_mode` for merging arrays. ([@leklund][])
+* [#4477](https://github.com/bbatsov/rubocop/issues/4477): Add new configuration directive: `inherit_mode` for merging arrays. ([@leklund][]). WARNING: To keep compatibility with older versions, follow [this hint](https://github.com/rubocop-hq/rubocop/issues/6021#issuecomment-399415948).
```

Comment:
This will break the build. The format is the changelog is strict and enforced by the CI. (the author has to be last thing on the line).

Reply:
@bbatsov could you check again pls?

Reply:
That's not the entry that's causing the breakage, btw - it's the one for https://github.com/rubocop-hq/rubocop/pull/5882 (which changes the bundled configuration and the inheritance behaviour for Include/Exclude).